src/App.jsx

```diff
@@ -14,70 +15,72 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [goodSelect, setGoogSelect] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
+      {goodSelect === null ? (
```

Comment:
```suggestion
      {!goodSelect ? (
```
you may write it shorter to check [falsy](https://developer.mozilla.org/en-US/docs/Glossary/Falsy) values